BibBuddy/app/src/main/java/de/bibbuddy/NoteModel.java

```diff
@@ -86,13 +86,14 @@ private String getDate(Long date) {
       Long noteId = note.getId();
       String modDate = getDate(note.getModDate());
       String name = "";
+      String text = note.getText();
       if (note.getType() == 0) {
         name = note.getName();
         name = Jsoup.parse(name).text();
         if (name.length() > 40) {
           name = name.substring(0, 35) + " ...";
         }
-        noteItemList.add(new NoteTextItem(modDate, name, noteId));
+        noteItemList.add(new NoteTextItem(modDate, name, text, noteId));
```

Comment:
You don't need to create a local variable for text. 
```suggestion
        noteItemList.add(new NoteTextItem(modDate, name, note.getText(), noteId));
```